exampleCourse/questions/element/dataframe/question.html

```diff
@@ -0,0 +1,25 @@
+<pl-question-panel>
+  <p>
+    The following examples below were created to show the capabilities of the <a href="http://prairielearn.readthedocs.io/en/latest/elements/#pl-dataframe"><code>pl-dataframe</code></a> element, which allows formatting and rendering various Python variables defined in the <code>server.py</code> on a question page.  For example, Pandas DataFrames can be set to a parameter and then rendered on the page a la Jupyter Notebook, or ordinary Python variables can be displayed in text format.
```

Comment:
This feels a bit like left over from the `pl-python-variable` component. I cleaned it a bit.

```suggestion
    The following examples below were created to show the capabilities of the <a href="http://prairielearn.readthedocs.io/en/latest/elements/#pl-dataframe"><code>pl-dataframe</code></a> element, which allows formatting and rendering of pandas data frames, e.g. <a href="https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.html"><code>pd.DataFrame()</code>, inside the <code>server.py</code> to be displayed on the question page. 
```